lib/ansible/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -667,12 +675,15 @@ def settings(module, proxmox, vmid, node, name, timeout, **kwargs):
         return False
 
 
-def create_vm(module, proxmox, vmid, newid, node, name, memory, cpu, cores, sockets, timeout, update, **kwargs):
+def create_vm(module, proxmox, vmid, newid, node, name, memory, cpu, cores, sockets, timeout, update, format, **kwargs):
     # Available only in PVE 4
     only_v4 = ['force', 'protection', 'skiplock']
 
     # valide clone parameters
-    valid_clone_params = ['format', 'full', 'pool', 'snapname', 'storage', 'target']
+    if module.params['full']:
+        valid_clone_params = ['format', 'full', 'pool', 'snapname', 'storage', 'target']
+    else:
+        valid_clone_params = ['full', 'pool', 'snapname', 'storage', 'target']
     clone_params = {}
     # Default args for vm. Note: -args option is for experts only. It allows you to pass arbitrary arguments to kvm.
     vm_args = "-serial unix:/var/run/qemu-server/{}.serial,server,nowait".format(vmid)
```

Comment:
Python 2.6 doesn't support non indexed format string, could they please be updated in this PR?

This may apply in a number of places where you have `{}`.

Reply:
Done.